source/reference/config/triggers.txt

```diff
@@ -155,6 +156,16 @@ configuration files:
        - ``"UPDATE"``
        - ``"REPLACE"``
        - ``"DELETE"``
+       - ``"CREATE_COLLECTION"``
+       - ``"MODIFY_COLLECTION"``
+       - ``"RENAME_COLLECTION"``
+       - ``"SHARD_COLLECTION"``
+       - ``"DROP_COLLECTION"``
+       - ``"RESHARD_COLLECTION"``
+       - ``"REFINE_COLLECTION_SHARD_KEY"``
+       - ``"CREATE_INDEXES"``
+       - ``"DROP_INDEXES"``
```

Comment:
[Q] do we want to add a blurb that these are `Database and Deployment only` won't fire/unsupported if watching at the collection level.